servers/rendering/shader_language.cpp

```diff
@@ -4076,13 +4076,16 @@ PropertyInfo ShaderLanguage::uniform_to_property_info(const ShaderNode::Uniform
 		case ShaderLanguage::TYPE_BOOL:
 			if (p_uniform.array_size > 0) {
 				pi.type = Variant::PACKED_INT32_ARRAY;
+				// TODO: Handle boolean representation.
```

Comment:
Tried doing this with a `boolean` type hint but it didn't work correctly, could use a `flags` hint but unsure if it's as useful, or if it will be translated correctly, will test later